README.md

```diff
@@ -42,6 +42,10 @@ A sample `docker-compose.yml` configuration:
       ports:
         - 3306
 
+    cron:
+      image: meanbee/magento:5.6-cli
+      command: bash -c "cron && tail -f /var/log/cron.log"
```

Comment:
Not something that needs to be fixed now, but a potential improvement for the future. We could write a wrapper script `run_cron.sh` that would start the rsyslog service, the cron service and tail the logs. That way:

1. rsyslog would only be running if needed
2. the command in docker-compose would only need to be `/run_cron.sh`, much simpler